modules/custom_operations/user_ie_extensions/tokenizer/python/tests/tokenizers_test.py

```diff
@@ -4,15 +4,20 @@
 
 import numpy as np
 import pytest
-from openvino import Core
-from openvino_tokenizers import (
-    convert_tokenizer,
-    pack_strings,
-    unpack_strings,
-)
+from openvino import Core, Tensor
+from openvino_tokenizers import convert_tokenizer
 from transformers import AutoTokenizer
 
 
+# Left these two methods for convenient transition from legay u8 representation to native string tensors
+# TODO: Remove the methods when transition is over
+def pack_strings(strings):
+    return strings
+
+def unpack_strings(strings):
+    return list(strings)
+
+
```

Comment:
Del all `(un)pack_strings` functions from tests. I think we also should delete them from `openvino_tokenizers.__init__.py` file.

Reply:
Do it in a separate commit please. I'm on vacation. This testing functionality -- I really found it useful to keep these two functions for debugging purposes as gates for all string tensors.